I card/Bhosle_Mahesh_21.jpg/id_card/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ID CARD</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css">
+</head>
+<body>
+    <!-- ID Card Wrapper -->
+    <div class="id-card-wrapper-front ">
+        <!-- Card Background Front Side -->
+        <div class="id-card-background">
+            <div class="right-big"></div>
+            <div class="top-right-small"></div>
+            <div class="left-bottom"></div>
+            <div class="right-small"></div>
+            <div class="right-bottom"></div>
+        </div>
+
+        <!-- Content of ID Card Front Side -->
+        <div class="id-card-content">
+            <!-- Logo -->
+            <div class="org-logo">
+                <img src="img/logo.png" alt="Logo">
+            </div>
+            <!-- ID Number -->
+            <div class="id-number">Id no : 123456</div>
+            
+            <!-- Photo -->
+            <div class="user-photo">
+                <img src="img/profile.jpg" alt="Photo">
+            </div>
+
+            <!-- Name -->
+             <div class="my-name"><span>BHOSLE</span><H1>MAHI</H1></div>
+
+
+            
+            
+            
+            <!-- Job Title/Grade/Class -->
+            <div class="id-title"><span>Jr.Software_Engineer</span> </div>
+
+            <!-- Organization/School/College Information -->
+             <div class="org-name">Apnasite It Services</div>
+            <div class="org-address">Kalewadi,Pune</div>
+        </div> 
+    </div>
+
+    <!-- ID Card Wrapper Back Side -->
+    <div class="id-card-wrapper-back">
+        <!-- Card Background Back Side -->
+        <div class="id-card-background">
+            <div class="top-right2"></div>
+            <div class="top-right-small2"></div>
+            <div class="bottom-left2"></div>
+            <div class="bottom-right2"></div>
+            <div class="mid-bottom1"></div>
+            <div class="mid-bottom2"></div>
+        </div>
+
+        <!-- Content of ID Card Back Side -->
+        <div class="id-card-content">
+            <div class="id-number2">id no: 123456</div>
+
+             <div class="user-name"><span>BHOSLE</span><H1>MAHI</H1></div>
+
+
+            <div class="qr-code">
+                <img src="img/qr.jpg" alt="Photo">
+            </div>
+
+            <!-- Emergency Contact Information -->
+            <div class="id-emergency-contact">
+                <div class="Email"><i class="fa-solid fa-envelope"></i>mbhosle061@gmail.com</div>
+                <div class="phone"><i class="fa-solid fa-phone"></i> +91 9665650029</div>
+            </div>
+
+            <!-- Blood Group -->
+            <div class="id-blood-group">
+                Blood Group:O+
+            </div>
+                <div class="dates">
```

Comment:
Remove this div class name dates, if you have any issue to set content of ID Back Card write separate CSS for each class eg. id-blood-group/id-dob/id-issue-date/id-expiry-date